edge-rtac/src/main/resources/domain/edge-rtac/features/rtac.feature

```diff
@@ -0,0 +1,156 @@
+    And match ['Available','Checked out'] contains call expectedData response.holdings,'status'
+    #delete created Items
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@DeleteItems')
+
+  Scenario: For a non-periodical/non-serial, return holdings and item information including availability for each instance UUID included in request
+#   1st instance
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostModeOfIssuance')
+    * def createInstanceResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostInstance')
+    * def firstInstanceId = createInstanceResponse.instanceEntityRequest.id
+    * def instanceId = firstInstanceId
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostServicePoint')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostLocation')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostHoldings')
+    * def createFirstItemResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostItem')
+    * def expectedFirstItemId = createFirstItemResponse.itemId
+#   2nd instance
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostModeOfIssuance')
+    * def createInstanceResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostInstance')
+    * def secondInstanceId = createInstanceResponse.instanceEntityRequest.id
+    * def instanceId = secondInstanceId
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostServicePoint')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostLocation')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostHoldings')
+    * def itemStatusName = 'Checked out'
+    * def createSecondItemResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostItem')
+    * def expectedSecondItemId = createSecondItemResponse.itemId
+
+    Given url edgeUrl
+    And path 'rtac?instanceIds=' + firstInstanceId + ',' + secondInstanceId
+    And param apikey = apikey
+    When method GET
+    Then status 200
+    And print response
+    And match response.instances.holdings.length == 2
+    And match [firstInstanceId,secondInstanceId] contains call expectedData response.instances.holdings,'instances'
+    And match [expectedFirstItemId,expectedSecondItemId] contains call expectedData response.instances.holdings,'holdings'
+    And match ['Available','Checked out'] contains call expectedData response.instances.holdings,'status'
+    #deleteItem
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@DeleteItems')
+
+  Scenario:For periodical/serial, return holdings and item information including availability for each instance UUID included in request WHEN &fullPeriodicals=true
```

Comment:
Please format these consistently. Some have a space after the colon (which I prefer) others do not.